content/BindingMethods/BindingMethods.facts.md

```diff
@@ -0,0 +1,50 @@
+### Facts for Prepared Statement with Binding Methods
+
+Parameter binding in a prepared statement is essential against SQL injections. The `bindValue()` and `bindParam()` methods bind variables or values in a prepared statement that contains placeholders as parameters such as a positional placeholder `?` or named placeholder `:variable_name`.
```

Comment:
As a learner, after reading this, I would ask myself `why would I use this over passing parameters on execute`? We need to answer that question. 

Also, SQL injection isn't just the reason why we use parameter binding. In fact it's not even the most common reason why we use binding parameter over passing a parameter on execute.

Let's rework this one.